package.json

```diff
@@ -39,8 +39,8 @@
     ]
   },
   "scripts": {
-    "build-css": "sass src/:src/ --load-path=.",
-    "watch-css": "yarn run build-css && sass src/ --watch --load-path=.",
+    "build-css": "sass src/:src/ --load-path=src/config/style --load-path=src/style --load-path=.",
```

Comment:
The load-path order here is crucial; if `src/config/style` is not first, then the `_variable`/`_style_overrides.scss` files outside of the portal configuration (which are required to compile SCSS without a configuration) will take precedence.